server/src/com/cloud/configuration/Config.java

```diff
@@ -672,6 +672,7 @@
             "86400",
             "The interval (in seconds) to wait before running the storage cleanup thread.",
             null),
+    StorageCleanupDelay("Advanced", StorageManager.class, Integer.class, "storage.cleanup.delay", "86400", "Determines how long (in seconds) to wait before actually expunging destroyed volumes. The default value = the default value of storage.cleanup.interval.", null),
```

Comment:
Please use the Configurable interface for configuration parameters instead. Refer to https://cwiki.apache.org/confluence/display/CLOUDSTACK/Configuration